### ISSUE TITLE
Hide meta fields in admin

### ISSUE THREAD

**Issue by nikola3244** [WP-for-Church/Sermon-Manager#191]
More details: https://wordpress.org/support/topic/feature-request-hide-input-fields/

>There are several input fields that we do not use (i.e. video embed code, video link, bulletin, etc…) It would be great if there was an option to hide these fields from editor side.
>
>We have team members we’re teaching to edit the sermons, and it would be easier for them if the fields they didn’t have to worry about weren’t even there, so they just have to enter in the fields they see.
>
>Not a big deal, but it would help streamline the process and produce less errors if we could just show the fields we needed.
>
>Thanks!

Note: #190 has to be done first

**Comment by scottdwest** [comment]
WordPress already has an interface to just hide the input files form the user display that are not used, could this be leveraged?

**Comment by nikola3244** [comment]
What WordPress does is that it hides a whole meta box, while this is about hiding specific fields inside a meta box. My idea is to create a cog or three dots in the top right of each meta box, for settings about hiding and showing the fields. But this is low priority for now - someone can jump in if they like :)
